CNV/olm-install.sh

```diff
@@ -0,0 +1,227 @@
+#!/bin/bash
+
+set -e
+
+# Optional file to set environment variable to non-default values instead of defining them on shell session level. File myenv.sh should be located in 
+# current directory 
+if [[ -f ./myenv.sh ]]; then 
+  echo ">>> myenv.sh file is present, setting some of the variables to non-default values"
+  . ./myenv.sh
+fi
+
+#Global namespace in OpenShift version 4.2 supposed to be openshift-marketplace 
+
+##################
+# Variables defining behavior of the script 
+# CUSTOM_APPREGISTRY - if true will use quay registry as application registry
+CUSTOM_APPREGISTRY=${CUSTOM_APPREGISTRY:-true}
+# NAMESPACED_SUBSCR - if true will create operator group explicitely specifying target namespaces that will be managed by operator otherwise it will set 
+#                     operator group globally 
+NAMESPACED_SUBSCR=${NAMESPACED_SUBSCR:-false}
+# WAIT_FOR_OBJECT_CREATION - Time in seconds for script to wait for some of the required objects to be created in Kubernetes. Time out will cause script to exit
+WAIT_FOR_OBJECT_CREATION=${WAIT_FOR_OBJECT_CREATION:-60}
+
+
+
+GLOBAL_NAMESPACE="${GLOBAL_NAMESPACE:-openshift-marketplace}"
+APP_REGISTRY="${APP_REGISTRY:-rh-osbs-operators}"
+PACKAGE="${PACKAGE:-kubevirt-hyperconverged}"
+TARGET_NAMESPACE="${TARGET_NAMESPACE:-openshift-cnv}"
+OPERATOR_NAME="${OPERATOR_NAME:-hco-operatorhub}"
+CHANNEL_VERSION="${CHANNEL_VERSION:-2.1.0}"
+
+
+if [[ ${CUSTOM_APPREGISTRY} ]]
+  then 
+
+    ####################
+    QUAY_USERNAME="${QUAY_USERNAME:-}"
+    QUAY_PASSWORD="${QUAY_PASSWORD:-}"
+
+    if [ -z "${QUAY_USERNAME}" ]; then
```

Comment:
Can you look at PR #141  and incorporate the changes there,

Prefer to have token variable, and do not want to be interactive.

Reply:
I have made the change to use either token or username/password . In both cases we still will do validation against Quay registry to make sure that token or ID are valid